packages/tests/src/ethereum_test_types/transaction_types.py

```diff
@@ -172,7 +172,7 @@ class TransactionGeneric(BaseModel, Generic[NumberBoundTypeVar]):
     """
 
     ty: NumberBoundTypeVar = Field(0, alias="type")  # type: ignore
-    chain_id: NumberBoundTypeVar = Field(
+    chain_id: NumberBoundTypeVar = Field( # type: ignore[assignment]
```

Comment:
Just missing a space for the CI. And added the `unused-ignore` element so `tox -e static` passes. 
```suggestion
    chain_id: NumberBoundTypeVar = Field(  # type: ignore[assignment, unused-ignore]
```